public/app/plugins/panel/graph3/module.tsx

```diff
@@ -154,6 +155,22 @@ export const plugin = new PanelPlugin<Options, GraphFieldConfig>(GraphPanel)
           defaultValue: { type: ScaleDistribution.Linear },
           shouldApply: f => f.type === FieldType.number,
           process: identityOverrideProcessor,
+        })
+        .addCustomEditor({
+          id: 'hideFrom',
+          name: 'Hide in area',
+          category: ['Series'],
+          path: 'hideFrom',
+          defaultValue: {
+            tooltip: false,
+            graph: false,
+            legend: false,
+          },
+          editor: SeriesConfigEditor,
+          override: SeriesConfigEditor,
+          shouldApply: () => true,
+          showIf: () => false,
```

Comment:
No need for this "hack". You can use the legitimate API introduced in https://github.com/grafana/grafana/pull/29879